src/ActionList.jsx

```diff
@@ -19,6 +20,31 @@ export default class ActionList extends Component {
 
   componentDidMount() {
     this.scrollToBottom(true);
+    if (!this.props.draggableActions) return;
+    const container = ReactDOM.findDOMNode(this.refs.rows);
```

Comment:
Sure, but it was introduced before and used in other places. Better let's keep it for another PR.